app/com/gu/floodgate/runningjob/RunningJobTable.scala

```diff
@@ -19,17 +20,19 @@ class RunningJobTable(protected val dynamoDB: AmazonDynamoDBAsyncClient, protect
   override protected def fromItem(item: Map[String, AttributeValue]): RunningJob =
     RunningJob(
       getItemAttributeValue(fields.ContentSourceId, item).getS,
-      getItemAttributeValue(fields.Progress, item).getN.asInstanceOf[Int], // why does toInt not work?
-      new DateTime(getItemAttributeValue(fields.StartTime, item).getS)
-    )
+      getItemAttributeValue(fields.DocumentsIndexed, item).getS.toInt,
```

Comment:
Storing an integer as an `S` is a shame. I think if you store it as an `N` then you might have to convert it to a `BigDecimal` and then an `Int`. That's what the DynamoDB Item API does.